cluster_loader/README.md

```diff
@@ -0,0 +1,42 @@
+# Cluster Loader
```

Comment:
Not to be overly picky but the below readme feels very disjointed. There are 5 "notes" which discuss additional variables and package requirements. IMHO I would rewrite it to include everything above line 31 as a "Requirements" section (cleaning it up to have a "Package requirements", "Variables", etc, etc) and then have your invoking be the python run_snafu ... down.

Reply:
done